### PR TITLE
add benchmarks for layer norm fused ops

### PR DESCRIPTION
**Pull request by liqiangxl**
add benchmarks for layer norm fused ops to test GPT-3 shapes.

### CODE REVIEW

**Review by naoyam — COMMENTED** [review]
LGTM. I'll let @kevinstephano give a stamp for confirming the problem sizes.

**Review by kevinstephano — APPROVED** [review]
LGTM

File: benchmark/layer_norm_fused.cpp [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+  auto tv13 = add(tv11, tv12);
+  auto tv14 = castOp(DataType::Half, tv13);
+  auto tv15 = castOp(DataType::Float, tv14);
+  auto tv16 = variance(tv15, {1}, false, false);
+  auto tv17 = broadcast(tv16, {false, true});
+  auto tv18 = sum(tv15, {1}, false);
+  auto tv19 = broadcast(tv18, {false, true});
+
+  nvfuser::Val* num_features =
+      IrBuilder::create<Double>(1, dtype = DataType::Double);
```
Comment (zasdfgbnm) [inline benchmark/layer_norm_fused.cpp:60]: What is this? I don't think this is C++ syntax.

Comment (liqiangxl) [reply benchmark/layer_norm_fused.cpp:60]: You're absolutely correct! This code was actually auto-translated from Python, hence the Pythonic style. Interestingly, it still functions correctly in C++ (looks like it just assign `DataType::Double` to `dtype` and then pass `dtype` to the function). As dtype isn't used after line 60, it doesn't impact the results. I'll submit a pull request to rectify this function call.

**Pull request merged**